### PR TITLE
Add 1.8 eol notification.

### PR DESCRIPTION
**Pull request by bianpengyuan**
[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/news/support/announcing-1.8-eol/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Support for Istio 1.8 ends on May 12th, 2021
+subtitle: Support Announcement
+description: Upcoming Istio 1.8 end of life announcement.
+publishdate: 2021-04-12
+aliases:
+    - /news/2021/announcing-1.8-eol
+---
+
+According to Istio's [support policy](/docs/releases/supported-releases#supported-releases/), LTS releases like 1.8 are supported for three months after the next LTS release. Since [1.9 was released on February 9th](/news/releases/1.8.x/announcing-1.9/), support for 1.8 will end on May 12th, 2021.
```
Comment (bianpengyuan) [inline content/en/news/support/announcing-1.8-eol/index.md:10]: @duderino @myidpt 

As @howardjohn suggested, I extended the support window by 3 days to make 1.8 also include the 5/11 fix. Does this make sense? The wording seems a bit weird though, I guess we'd better make it clear there is a three days extension?

**Comment by bianpengyuan** [comment]
cc @GregHanson

File: content/en/news/support/announcing-1.8-eol/index.md (hunk 1 above)
Comment (myidpt) [reply content/en/news/support/announcing-1.8-eol/index.md]: That sounds good to me.

**Review by craigbox — CHANGES_REQUESTED** [review]
#8758 removes the use of the term "LTS" in favor of "minor release", which has been updated in https://istio.io/latest/about/supported-releases/.

s/LTS/minor/ might be enough to update this.

**Comment by bianpengyuan** [comment]
@craigbox Done. ptal thanks!

**Comment by ericvn** [comment]
LGTM

**Comment by bianpengyuan** [comment]
@ericvn @craigbox ping on this. I'd like to publish this today if possible. Thanks a lot!

**Review by ericvn — APPROVED** [review]

**Pull request merged**